test/factories/block.rb

```diff
@@ -5,8 +5,7 @@
     number { 10 }
     parent_hash { "0xcba2d1a70602a1def80efbd59629c37a9d6c36f9de7a8ed6d1ca4f76389365e1" }
     nonce { 1757392074788233522 }
-    # FIXME: the timestamp in blocks table is in milliseconds but here is in seconds
-    timestamp { Faker::Time.between(from: 2.days.ago, to: Date.today).to_i }
+    timestamp { Faker::Time.between(from: 2.days.ago, to: Date.today).to_i * 1000 }
```

Comment:
Here should use a fixed value. You can get number 10 block's timestamp from vcr files. It's 1570197479041.